### ISSUE TITLE
Feature request: Start paused

### ISSUE THREAD

**Issue by hithwen** [steeve/plugin.video.pulsar#115]
Some times it takes a few minutes for an episode to buffer before it can start playing. In that time you might go away and do other stuff, then it suddenly starts playing while you're not watching. It would be nice to have a config option to start playing in pause.

**Comment by sjimmie** [comment]
Excellent idea!  +1

**Comment by davux** [comment]
Great idea.

Not sure about the best implementation:
- Context menu option "Start paused"
- Separate, generic "Start paused" plugin (because maybe it makes sense in other situations?)
- Change in workflow: when opening a stream, Pulsar first enters "play" mode (to the eyes of Kodi) before starting to download the initial buffer. This way, as a user you can just pause the video while the buffer fills up and there's no special feature needed.
